### PR TITLE
Adding the Songhay Language Pack to the new Repo

### PR DESCRIPTION
**Pull request by Afudu**
Hello Team,

Please add the Songhay pack to the new repository.

Many thanks in advance

### CODE REVIEW

**Comment by Afudu** [comment]
closed by mistake

File: addons/languages/songhay/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```
Comment (menny) [inline addons/languages/songhay/apk/build.gradle:1]: Should be set to `false`, until i have uploaded a signed version to the Play Store. Please change

File: addons/languages/songhay/pack/dictionary/prebuilt/additionals.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (menny) [inline addons/languages/songhay/pack/dictionary/prebuilt/additionals.xml:1]: I guess, you don't need this file?

**Review by menny — CHANGES_REQUESTED** [review]
Good job!
Just a couple of comments, and we're good to go

File: addons/languages/songhay/apk/build.gradle (hunk 1 above)
Comment (Afudu) [reply addons/languages/songhay/apk/build.gradle]: Thanks, Menny.  I changed it to false.

File: addons/languages/songhay/pack/dictionary/prebuilt/additionals.xml (hunk 2 above)
Comment (Afudu) [reply addons/languages/songhay/pack/dictionary/prebuilt/additionals.xml:1]: Thanks. I updated it and changed some of the words to to Songhay pronunciation.

**Review by menny — APPROVED** [review]

File: addons/languages/songhay/apk/build.gradle [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = false
```
Comment (menny) [inline addons/languages/songhay/apk/build.gradle:1]: Please also add another line for the reason of not publishing:
```
ext.notPublishingReason = "Initial version has not been published yet"
```

**Review by menny — CHANGES_REQUESTED** [review]

File: addons/languages/songhay/apk/build.gradle (hunk 3 above)
Comment (Afudu) [reply addons/languages/songhay/apk/build.gradle:1]: Thank you. The line has been added.

**Review by menny — DISMISSED** [review]

**Review by menny — APPROVED** [review]

**Comment by menny** [comment]
Thank you

**Comment by menny** [comment]
Can you attach two screenshots of the new layout? Portrait and landscape

**Pull request merged**